archunit/src/test/java/com/tngtech/archunit/testutil/JavaCallQuery.java

```diff
@@ -0,0 +1,63 @@
+package com.tngtech.archunit.testutil;
+
+import java.util.Set;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import com.tngtech.archunit.core.domain.JavaCodeUnit;
+import com.tngtech.archunit.core.domain.JavaMethod;
+import com.tngtech.archunit.core.domain.JavaMethodCall;
+
+public class JavaCallQuery {
+
+    private FluentIterable<JavaMethodCall> calls;
+
+    private JavaCallQuery(Set<JavaMethodCall> calls) {
+        this.calls = FluentIterable.from(calls);
+    }
+
+    public static JavaCallQuery methodCallTo(final JavaMethod method) {
+        return new JavaCallQuery(method.getCallsOfSelf()).that(hasTarget(method));
```

Comment:
Isn't `that(hasTarget(method))` redundant to `method.getCallsOfSelf()`?

Reply:
Yes, I think so :joy: Adjusted it!!